### DISCUSSION TITLE
Cannot find connected teleport-kube-agent

### DISCUSSION THREAD

**Discussion by champs** [gravitational/teleport#8887] [Q&A] [answered]
Hello there,

I have newly setup teleport cluster and another k8s remote cluster. I followed [Kubernetes Access Multiple Clusters](https://goteleport.com/docs/kubernetes-access/guides/multiple-clusters/) doc and got agent installed.

The logs teleport-agent seem to be connected 
```
2021-11-06T07:04:40Z INFO [KUBERNETE] Connected. addr:<REDACTED>:52106 remote-addr:<REDEACTED>:3024 leaseID:14 target:<FQDN>:3024 reversetunnel/agent.go:354
```

but the `tsh kube ls` return an empty list
```
tsh kube ls
Kube Cluster Name Selected 
----------------- -------- 

```
I am logged in as admin with full permission to view Kube obj.  I don't see any Error on teleport-cluster / or agent, both of them seem to connect to each other. Can someone please help me understand how to debug this issue?

**Comment by webvictim** [comment, marked as answer]
Do any of the roles you have (see the output of `tsh status`, and look them up with `tctl get role/<name>` on the auth server) have `kubernetes_labels` set? If none of them do, I think it defaults to "no permissions".
